libpod/container_internal.go

```diff
@@ -932,6 +925,8 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		}
 		if !MountExists(g.Mounts(), dstPath) {
 			g.AddMount(newMount)
+		} else {
+			logrus.Warnf("mount being overwritten by user %q", dstPath)
```

Comment:
Can you rephrase this to make it a little more clear? Maybe "User mount overriding libpod mount at %s"?

Reply:
fixed